cmd/login.go

```diff
@@ -103,16 +103,27 @@ func runOCMWorkspaceContainer(
 	serviceRef string,
 	isOcmLoginOnly bool) {
 	envVarOcmUser := fmt.Sprintf("OCM_USER=%s", viper.GetString("ocUser"))
-	envVarOcmToken := fmt.Sprintf("OCM_TOKEN=%s", viper.GetString("ocmToken"))
+	// envVarOcmToken := fmt.Sprintf("OCM_TOKEN=%s", viper.GetString("ocmToken"))
 	envVarCluster := fmt.Sprintf("OCM_CLUSTER=%s", ocmCluster)
 	envVarIsOCMLoginOnly := fmt.Sprintf("IS_OCM_LOGIN_ONLY=%v", isOcmLoginOnly)
-	userHome := fmt.Sprintf("/home/%s", viper.GetString("ocUser"))
+	userHome := fmt.Sprintf("%s", viper.GetString("userHome"))
```

Comment:
Are you going to require the `userHome` config? I'm fine with it but the reason I didn't require it is because I wanted it to maintain as little config as it can. 

I'll also create a Todo for automatic config validation as the missing configs will cause failures badly.

Reply:
userHome can be different for each user. In my case ocUser != userHome hence I had errors trying to run the login script. Can we please have it as config value for now?